firebase/firebase-cpp-sdk
# Auth Integration Test Project for tvOS

Added a integration_test_tvos target to the auth integration test xcode project.  

Additonally added a SKIP_TEST_ON_TVOS to `testing/test_framework/src/firebase_test_framework.h` and used it to skip the phone auth test on tvOS.  Phone Auth is not supported on thhat platform.

To test this,

Make sure to have a valid GoogleService-Info.plist.
pod install
open integration_test.xcworkspace
Drag and drop firebase.xcframework and firebase_remote_config.xcframework
